### ISSUE TITLE
Get to a working eth binary for Android

### ISSUE THREAD

**Issue by bobsummerwill** [doublethinkco/webthree-umbrella-cross#35]

**Comment by bobsummerwill** [comment]
Hey @anthony-cros!    See https://github.com/bq/aquaris-E5/tree/aquaris-E5-ubuntu-master.

That has the following step listed ...

    sudo apt-get install gcc-arm-linux-androideabi

So if we can get gcc-arm-linux-gnueabi working then it looks like adding specifically Android-targeted binaries would be easy enough from there.

**Comment by anthony-cros** [comment]
Good find, though i suspect we'll run into the same issue as with [the other binaries](https://github.com/doublethinkco/webthree-umbrella-cross/issues/42#issuecomment-163791171)!

**Comment by bobsummerwill** [comment]
Yes, we will, because it will be another ARMv7 ABI one, I think.

**Comment by anthony-cros** [comment]
it shouldn't be too hard to adapt the scripts to point to that binary instead of the others though, so at least once the ARMv7 issue is solved (via -fPIC), we should that that one "for free"

**Comment by bobsummerwill** [comment]
Yes, that should be easy enough.
